try {
  foo();
  console.log('one');
} catch (e) {
  console.log('two');
} finally {
  console.log('three');
}
